articles/libraries/auth0js/v7/index.md

```diff
@@ -12,6 +12,10 @@ This document covers an outdated version of auth0.js. We recommend you to <a hre
 
 Auth0.js is a client-side library for [Auth0](http://auth0.com), for use in your web apps. It allows you to trigger the authentication process and parse the [JSON Web Token](http://openid.net/specs/draft-jones-json-web-token-07.html) (JWT) with just the Auth0 `clientID`. Once you have the JWT, you can use it to authenticate requests to your HTTP API and validate the JWT in your server-side logic with the `clientSecret`.
 
+::: note
+Check out the [Auth0.js repository](https://github.com/auth0/auth0.js) on GitHub.
```

Comment:
I think that this should point here https://github.com/auth0/auth0.js/tree/v7

cc @jeffreylees